app/celery/service_callback_tasks.py

```diff
@@ -17,44 +17,78 @@
 
 @notify_celery.task(bind=True, name="send-delivery-status", max_retries=5, default_retry_delay=300)
 @statsd(namespace="tasks")
-def send_delivery_status_to_service(self, notification_id,
-                                    encrypted_status_update
-                                    ):
-    try:
-        status_update = encryption.decrypt(encrypted_status_update)
+def send_delivery_status_to_service(
+    self, notification_id, encrypted_status_update
+):
+    status_update = encryption.decrypt(encrypted_status_update)
+
+    data = {
+        "notification_id": str(notification_id),
```

Comment:
Changing id to notification_id could affect the clients using this data. I don't think we want to change the name of the key.

Reply:
Good point, I will change it back.